src/Pos/Explorer/Web/Doc.hs

```diff
@@ -103,6 +128,19 @@ instance ToSample ExplorerError where
 
 {-
 
+data CHashSearchResult
```

Comment:
@ksaric Could you remove this? I see it's defined in another module, perhaps you just forgot to remove this comment